.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name : Release
+
+on:
+  push:
+    tags:
```

Comment:
@rsbh I think we should do it on release, right?

Reply:
i am thinking to automate release on tag creating. locally learn will create git tag and push that

Reply:
@ravisuhag  Yeah if we put on release we need to manually make a github release and then the publish will be triggered but we want to automate github release also . so when we push tags we publish the changed packages and then make a github release based on tags.